installer/manifests/keptn/core-distributors.yaml

```diff
@@ -140,7 +139,7 @@ spec:
             - name: PUBSUB_URL
               value: 'nats://keptn-nats-cluster'
             - name: PUBSUB_TOPIC
-              value: 'sh.keptn.event.tests.finished'
```

Comment:
IMHO the `sh.keptn.event.tests.finished` is correct?

In go-utils we have:
```go
// TestsFinishedEventType is a CloudEvent for indicating that tests have finished
const TestsFinishedEventType = "sh.keptn.event.tests.finished"

// TestFinishedEventType_0_5_0_Compatible is a CloudEvent for indicating that tests have finished
const TestFinishedEventType_0_5_0_Compatible = "sh.keptn.events.tests-finished"
```

Reply:
I removed the `TestFinishedEventType_0_5_0_Compatible` as we want no backward support of changed event types. Thus, I changed the `sh.keptn.event.tests.finished` event to  sh.keptn.events.tests-finished` , which follows the naming convention of others. 
```
// TestsFinishedEventType is a CloudEvent for indicating that tests have finished
const TestsFinishedEventType = "sh.keptn.events.tests-finished"
```

Reply:
Thanks for clarification.